app/src/routes/v1/acronyms.js

```diff
@@ -6,6 +6,34 @@ const log = require('npmlog');
 const Problem = require('api-problem');
 
 const acronyms = require('../../components/acronyms');
+const permissionHelpers = require('../../components/permissionHelpers');
+
+/** fetches the acronym details */
+acronymsRouter.get('/:appAcronym', [
+], async (req, res) => {
```

Comment:
If we aren't doing any validation on this, we don't need to stick in an empty array as an extra middleware object.

Reply:
Changed